tpl/gcp/bin/concourse-deploy.sh.tt

```diff
@@ -8,4 +8,4 @@ bosh -e <%= @values['bosh']['name'] %> upload-release https://github.com/concour
 bosh -e <%= @values['bosh']['name'] %> upload-release https://github.com/concourse/concourse/releases/download/v3.4.1/garden-runc-1.6.0.tgz
 
 # Deploy Concourse
```

Comment:
@vshatravenko We can do better than that
first we need a helper kite for generator strong password
also we should save generated password.

vault token is another token but we could save it also in the a vars files
then we can have a bosh vars file full of passwords, file we can save later in vault or something.